README.md

```diff
@@ -1,6 +1,192 @@
 # Shrinkwrap
 
-ShrinkWrap is a library to create archives from Java, which powers the [Arquillian](https://github.com/arquillian)
-deployment mechanism. Arquillian in turn is a library.
+ShrinkWrap is a Java library to create Java archives such as JARs, WARs, EARs and RARs.
+This is especially useful for integration testing, where you need to deploy specific configurations of your applications, such as [Arquillian](https://github.com/arquillian).
+
+## Requirements
+
+- **JDK**: Version 8 or newer.
+- **Maven**: Version 2.2.0 or newer.
```

Comment:
Just out of curiosity, why 2.2.0?

Reply:
That is the minimum version I found in the Maven enforcer plugin.

Reply:
Have you tried building it with Maven 2.2.0? I'm curious if it will even work